staging/src/k8s.io/client-go/transport/config.go

```diff
@@ -132,5 +132,5 @@ type TLSConfig struct {
 	// To use only http/1.1, set to ["http/1.1"].
 	NextProtos []string
 
-	GetCert func() (*tls.Certificate, error) // Callback that returns a TLS client certificate. CertData, CertFile, KeyData and KeyFile supercede this field.
+	GetCert func() (*tls.Certificate, error) // Callback that returns a TLS client certificate. Supercedes CertData, CertFile, KeyData and KeyFile
```

Comment:
@deads2k its worth noting that this change is a consequence of keeping the filenames around, and it could break user-facing stuff potentially